lib/auth/auth.go

```diff
@@ -2947,22 +2950,60 @@ func (a *Server) GetAccessCapabilities(ctx context.Context, req types.AccessCapa
 	return caps, nil
 }
 
-// calculateMaxAccessTTL determines the maximum allowable TTL for a given access request
-// based on the MaxSessionTTLs of the roles being requested (a access request's life cannot
-// exceed the smallest allowable MaxSessionTTL value of the roles that it requests).
-func (a *Server) calculateMaxAccessTTL(ctx context.Context, req types.AccessRequest) (time.Duration, error) {
-	minTTL := defaults.MaxAccessDuration
-	for _, roleName := range req.GetRoles() {
+// resourceTTL calculates the TTL for the types.AccessRequest resource.
+func (a *Server) resourceTTL(ctx context.Context, identityExpires time.Time, r types.AccessRequest) (time.Duration, error) {
+	// If no expiration provided, use default (1 hour).
```

Comment:
I would just remove the comment here. It's pretty obvious that you're defaulting to `defaults.PendingAccessDuration`, and if we ever change the default you can almost guarantee that we'll forget to update the comment. IMO having a comment that is wrong is worse than no comment at all.